assets.json

```diff
@@ -39,6 +39,40 @@
             }
         ]
     },
+    {
+        "name": "Fuel",
+        "symbol": "FUEL",
+        "icon": "images/fuel.svg",
+        "networks": [
+            {
+                "type": "ethereum",
+                "chain": "sepolia",
+                "address": "0x675b68aa4d9c2d3bb3f0397048e62e6b7192079c",
```

Comment:
Just realized this address isn’t correct on Sepolia. Doesn’t look like anything is deployed there & it matched the Mainnet address.